tutorials/websocket_tutorial/websocketexample.py

```diff
@@ -145,7 +145,6 @@ def make_cube():
                 OptionModelingCmdReq(
                     cmd=ModelingCmd(
                         OptionExtrude(
-                            cap=True,
```

Comment:
this is not a param https://docs.rs/kittycad-modeling-cmds/latest/kittycad_modeling_cmds/struct.Extrude.html